mlflow/tensorflow.py

```diff
@@ -212,11 +218,15 @@ def load_model(model_uri, tf_sess):
                       `Referencing Artifacts <https://www.mlflow.org/docs/latest/tracking.html#
                       artifact-locations>`_.
 
+    For TensorFlow versions < 2.0.0:
```

Comment:
It might be clearer to describe the difference in behavior between <2.0.0 and >=2.0.0 in the `tf_sess` param doc itself. Namely, it'd be good to clarify that `tf_sess` is required for TF<2.0.0 and unused otherwise